### DISCUSSION TITLE
atac_singlecell.csv does not have passed_filters

### DISCUSSION THREAD

**Discussion by rhart604** [stuart-lab/signac#709] [Q&A] [answered]
My atac_singlecell.csv file does not have a passed_filters field. Only is_cell_barcode and excluded_reason fields are found. I used cellranger-arc v. 2.0.0. Is it appropriate to filter instead on excluded_reason == 0?

**Comment by timoast** [comment, marked as answer]
The cellranger arc output contains several single-cell metrics, see the documentation on the 10x website: https://support.10xgenomics.com/single-cell-multiome-atac-gex/software/pipelines/latest/output/per_barcode_metrics

In this case, `passed_filters` from cellranger-atac would correspond to `atac_fragments` in cellranger-arc
